### DISCUSSION TITLE
Custom dataset consisting only of syntheised images

### DISCUSSION THREAD

**Discussion by AminSeffo** [facebookresearch/detectron2#3873] [Q&A]
Hey everyone,

I would like to know if anyone here trains his custom dataset only with synthesized images without any real images.
I did it already and used the same config, which is already used in Detectorn2 tutorial but the result is not sufficient. Recently I found a paper [https://www.arxiv-vanity.com/papers/1710.10710/](url) that presents a new method to boost performance during training by freezing the **feature extractor** in all layers.
So do someone know, how can I do this in detectron2 or has other methods to improve the performance?

Any answer is very appreciated.

**Comment by zhanghang1989** [comment]
You may freeze different number of stages for the backbone. If you'd like to freeze the entire feature extractor, set it to 5 https://github.com/facebookresearch/detectron2/blob/main/detectron2/config/defaults.py#L137

**Reply by AminSeffo** [reply]
@zhanghang1989 thank you very much for your response.  Sorry because the above link doesn't work, here is the correct one: https://arxiv.org/abs/1710.10710
Would this also mean that the feature extractor’s weights are frozen and they can't adapt during the training?

**Reply by zhanghang1989** [reply]
Yes, frozen weights won't be adapted
